uint8[] index # from 0
int16[] angles # dynamixel: 15bit
